src/config.js

```diff
@@ -12,29 +14,34 @@ type ApplyConfigToArgvParams = {|
   argv: Object,
   configObject: Object,
   defaultValues: Object,
+  configFileName?: string,
```

Comment:
Yes, if I don't make this optional, I'll have to add a file path to all the tests.